### DISCUSSION TITLE
how can i simulate requests from the frontend that send files in formData

### DISCUSSION THREAD

**Discussion by joaomelo** [usebruno/bruno#364] [Q&A]
Great project. We are excited to use Bruno and replace our current tooling based on Postman. But we got stuck in one of our tests.

How can we emulate a front-end request that would send a PDF file as part of a formData?

We tried to use the Node fs.readFileSync function to load the file content from the filesystem as an env var to be consumed by the request, but Bruno considered both fs and readFileSync "undefined".

In Postman today we just load the file in the IDE GUI.

![MicrosoftTeams-image](https://github.com/usebruno/bruno/assets/5727050/4654815b-15b5-4d03-a020-07ca863b21f3)

Is there a way to solve that? Thank you very much for your help.

**Comment by jaktestowac** [comment]
Currently not supported, but hope it will be soon available see discussion here: 
https://github.com/usebruno/bruno/pull/258

**Comment by helloanoop** [comment]
@joaomelo We have fs access enabled now. See https://github.com/usebruno/bruno/issues/306

**Reply by joaomelo** [reply]
You folks are fast and furious! Thank you very much.

**Reply by helloanoop** [reply]
@joaomelo checkout https://github.com/usebruno/bruno/discussions/385#discussioncomment-7196674 this has what you are looking for.

Feel free to post your script too if you feel others can find it helpful.
